frontends/p4/fromv1.0/programStructure.cpp

```diff
@@ -285,12 +285,6 @@ class ProgramStructure::FixupExtern : public Modifier {
 };
 
 void ProgramStructure::createExterns() {
-    for (auto r : registers) {
```

Comment:
Ok, they generate declarations, but the declarations will be removed as dead code, as they are not referenced anywhere. 

Also, if you remove this error message at least one test will fail.